resources/kcp/charts/kyma-environment-broker/values.yaml

```diff
@@ -5,16 +5,16 @@ global:
       version: "PR-2570"
     kyma_environments_cleanup_job:
       dir:
-      version: "PR-2540"
+      version: "PR-2476"
     kyma_environments_subaccount_cleanup_job:
       dir:
-      version: "PR-2562"
+      version: "PR-2476"
     kyma_environment_trial_cleanup_job:
       dir:
-      version: "PR-2554"
+      version: "PR-2476"
     kyma_environment_deprovision_retrigger_job:
       dir:
-      version: "PR-2554"
+      version: "PR-2476"
```

Comment:
Why do you change the image tag (ref to PR name) in the image? If creation of binary image for subaccount cleanup job fail the job will fail. Job works fine with current image.